spec/spec_helper.rb

```diff
@@ -151,6 +151,17 @@ def enable_chrome_headless_downloads(driver)
     Warren.handler.disable!
   end
 
+  config.around(:each, accessioning_enabled: true) do |ex|
```

Comment:
There are a few messy bits to set up, so I've pulled them into an rspec configuration.

Reply:
Could you put a short comment, something like:
```
# For tests with the accessioning_enabled flag, do:
# setup: set the accession folder to a test folder and turn on accessioning
# teardown: turn accessioning off again if it was off previously.
```

...if that is accurate?
What is the accession folder for exactly and why don't you reset that in the teardown?

Reply:
It sets the configuration it loads the config from. I did actually try to reset it, but it seemed to blow away a bit more state than expected, and broke a load of other tests.